openstack/clientconfig/requests.go

```diff
@@ -18,6 +19,14 @@ type ClientOpts struct {
 
 	// EnvPrefix allows a custom environment variable prefix to be used.
 	EnvPrefix string
+
+	// AuthType specifies the type of authentication to use.
+	// By default, this is "password".
+	AuthType string
+
+	// Auth defines the authentication information needed to
+	// authenticate to a cloud when clouds.yaml isn't used.
+	Auth *Auth
```

Comment:
How about `AuthInfo` for this (both these, I guess)?

Reply:
Yes, makes sense.